### ISSUE TITLE
Electron DOMException

### ISSUE THREAD

**Issue by imohuan** [im-oxygen/libaria2-ts#1]
I don't know much about electron because libaria2-ts works in my browser, but when I put it in electron, I get an error.

``` javascript
rpc.ts:62 Uncaught (in promise) DOMException: Failed to construct 'WebSocket': The subprotocol '[object Object]' is invalid.
    at new Client (http://localhost:4000/node_modules/.vite/libaria2-ts.js?v=f0deda4e:1342:18)
    at init (http://localhost:4000/src/renderer/App.vue:7:21)
    at setup (http://localhost:4000/src/renderer/App.vue:15:5)
    at callWithErrorHandling (http://localhost:4000/node_modules/.vite/chunk-WIAEGI5J.js?v=f0deda4e:1020:18)
    at setupStatefulComponent (http://localhost:4000/node_modules/.vite/chunk-WIAEGI5J.js?v=f0deda4e:3211:25)
    at setupComponent (http://localhost:4000/node_modules/.vite/chunk-WIAEGI5J.js?v=f0deda4e:3178:36)
    at mountComponent (http://localhost:4000/node_modules/.vite/vue.js?v=f0deda4e:2044:5)
    at processComponent (http://localhost:4000/node_modules/.vite/vue.js?v=f0deda4e:2023:9)
    at patch (http://localhost:4000/node_modules/.vite/vue.js?v=f0deda4e:1742:11)
    at render2 (http://localhost:4000/node_modules/.vite/vue.js?v=f0deda4e:2554:7)
```
How should I modify that to use in Electron?

**Comment by hydrati** [comment]
@imohuan  please show your code

**Comment by imohuan** [comment]
I wrote in the Vue3+vite of the Electron render process
``` javascript
<template>
  <div>Hello</div>
</template>

<script lang="ts">
import { defineComponent } from "vue";
import { WebSocket } from "libaria2-ts";

export default defineComponent({
  setup() {
    console.log(WebSocket);
    const aria2 = new WebSocket.Client({
      host: "localhost",
      port: 6800,
      path: "/jsonrpc",
      auth: {}
    });

    aria2.getVersion().then((res) => {
      console.log(res);
    });

    return {};
  }
});
</script>

<style scoped></style>

```

**Comment by imohuan** [comment]
If you run it directly in the browser, it's perfect

**Comment by hydrati** [comment]
@imohuan did u enable `nodeintegration`

**Comment by hydrati** [comment]
@imohuan fixed in 1.0.708

**Comment by imohuan** [comment]
I opened nodeintegration, and you will know that's not the reason by reading the wrong newspaper!
